### PR TITLE
Updated minio images, add cronjob to set expiry to minio errors buckets

### PR DESCRIPTION
**Pull request by nutellinoit**
As per title.

This change is needed to expire old errors like we expire logs on elasticsearch after a set time.

### CODE REVIEW

File: katalog/logging-operated/minio/cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2022 SIGHUP s.r.l All rights reserved.
```
Comment (ralgozino) [inline katalog/logging-operated/minio/cronjob.yaml:1]:
```suggestion
# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
```

File: katalog/logging-operated/minio/cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+              envFrom:
+              - secretRef:
+                  name: minio-credentials
+              image: minio/mc
+              imagePullPolicy: IfNotPresent
+              securityContext:
+                allowPrivilegeEscalation: false
+          restartPolicy: OnFailure
+  schedule: "0 * * * *"
```
Comment (ralgozino) [inline katalog/logging-operated/minio/cronjob.yaml:46]: missing new line

**Review by ralgozino — CHANGES_REQUESTED** [review]
minor nitpicks.

Can we document somewhere about these expiring rules (I guess in the `operated` readme)?

**Comment by nutellinoit** [comment]
> Can we document somewhere about these expiring rules (I guess in the `operated` readme)?

Yes!

**Review by ralgozino — APPROVED** [review]
LGTM! 🚢

**Pull request merged**